spec/features/admin/facilities/index_spec.rb

```diff
@@ -73,4 +74,15 @@
 
     expect(page).to have_content(unassociated_facility02.name)
   end
+
+  it "verify manage section overlay for the given permission" do
```

Comment:
Can we update the description to something like `it "renders only allowed tabs for the given permissions"`? This will read a bit more naturally.